### DISCUSSION TITLE
Add the "Start new build ->" button to the builds screen

### DISCUSSION THREAD

**Discussion by mattlennon3** [codemagic-ci-cd/codemagic-docs#1760] [Ideas]
### Use case

```Text
I most often keep the builds screen open, to monitor build progress. If I need to trigger a new one, I have to navigate to apps and then start a build, then navigate back to the builds screen. 

It seems like it'd be more intuitive to start a build from the builds screen.
```

### Idea

```Text
Pretty much as described in the use case. I would like a button to start builds from the same place I monitor them. It would open the same modal as on the apps screen to select the branch and workflow.
```

**Comment by Ycoi** [comment]
Thanks for the idea @mattlennon3 ! You can actually already trigger new builds from the builds screen by clicking the three dots icon of a build. So just select a build of an app you want to trigger a build for, click the three dots icon and  select "Start new build”. Does this solve it for you?

I however can see that this is not obvious and it’s easy to miss. So having a separate button for this might make sense. I’ll discuss this  with the team and we’ll see what we can do.

Thanks again for the feedback! If you have any questions or more ideas don’t hesitate to post them here.
